lib/MusicBrainz/Server/Data/Statistics.pm

```diff
@@ -995,13 +995,16 @@ my %stats = (
         CALC => sub {
             my ($self, $sql) = @_;
 
-            my $data = $sql->select_list_of_lists(
-                q{SELECT COALESCE(l.iso_code_3::text, 'null'), COUNT(r.gid) AS count
-                FROM release r FULL OUTER JOIN language l
-                    ON r.language=l.id
-                WHERE l.iso_code_2t IS NOT NULL OR l.frequency > 0
-                GROUP BY l.iso_code_3},
-            );
+            my $data = $sql->select_list_of_lists(<<~'SQL');
+                         SELECT COALESCE(l.iso_code_3::text, 'null'),
+                                COUNT(r.gid) AS count
+                           FROM release r
+                FULL OUTER JOIN language l ON r.language=l.id
+                          WHERE l.iso_code_2t IS NOT NULL
+                             OR l.frequency > 0
+                             OR l.id IS NULL
```

Comment:
Among the ten lines of changed code in the first commit <https://github.com/metabrainz/musicbrainz-server/pull/2572/commits/0db6f1543444c6d9deabc008db887218db6c442a>, it seems to be the only one related to the commit message. It seems worth splitting coding style changes off from this commit.